packages/server/src/migrations/schema/v98.ts

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright Orangebot, Inc. and Medplum contributors
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * This is a generated file
+ * Do not edit manually.
+ */
+
+import { PoolClient } from 'pg';
+import * as fns from '../migrate-functions';
+
+// prettier-ignore
+export async function run(client: PoolClient): Promise<void> {
+  const results: { name: string; durationMs: number }[] = []
+  await fns.query(client, results, `CREATE TABLE IF NOT EXISTS "ConceptMapping" (
+    "id" BIGSERIAL PRIMARY KEY,
+    "conceptMap" UUID NOT NULL,
+    "sourceSystem" TEXT NOT NULL,
```

Comment:
I think it makes sense to create a separate `ConceptSystem` table or something and put a FK (e.g., `sourceCodeId`).  this way we can avoid having millions/billions entries of "http://snomed.info/sct").  

Much lower disk space, smaller indices, faster query performance, etc.